shoov/modules/custom/shoov_ci_build/shoov_ci_build.views.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file
+ * integration with the views module.
+ */
+
+/**
+ * Implements hook_views_data().
+ */
+function shoov_ci_build_views_data() {
+  $data['message']['trigger'] = array(
+    'title' => t('Trigger CI Build message'),
+    'help' => t('Link to send the message to nodejs server.'),
+    'field' => array(
+      'handler' => 'shoov_ci_build_trigger_button',
+    ),
+  );
+  return $data;
+}
+
+/**
+ * Implements hook_views_default_views_alter().
+ *
+ * Add link field to the view of messages to send the item to the nodejs server.
```

Comment:
do we really need an alter -- as we know trigger should only appear for CI build items. Anyway, to make it simpler, lets add the trigger only on this view

![gizra_tami4-shoov___shoov-3](https://cloud.githubusercontent.com/assets/125707/9703429/59b78330-548c-11e5-8fbb-cc0997b8f525.jpg)